_projects/nested-languages.md

```diff
@@ -24,16 +24,68 @@ tags:
   - Core
   - Bears
 ---
-Multiple programming languages can coexist in a single source file. coala would
-benefit from the ability to determine the programming language of each part of
-a file. If we have e.g. a PHP file, it is likely that we'd have chunks of html
-inside a few of them. We would like coala to be able to still run meaningful
-analysis on those files.
+Generally, a single source file contains code for exactly one programming language.
+But multiple programming languages can also co-exist in a single source file.
+If we have e.g. a PHP file, it is likely that we'd have
+chunks of HTML inside a few of them. We would like coala to be able to still
+run meaningful analysis on those files.
 Other examples include:
 
-* awk inside bash
-* JSON inside Javascript
-* HTML, LaTeX or others inside Jinja2
+* Web templating - languages such as PHP or JSP files mix code into HTML.
+* reStructuredText specify different language for codeblock, means we have a
+block of code which is written in some particular language and it is embedded
+inside reStructuredText syntax.
+
+rstcheckBear of coala checks the syntax of reStructuredText and code blocks
```

Comment:
shouldn't `rstcheckBear` be replaced by `RSTcheckBear` ?
Refer RSTcheckBear [class](https://github.com/coala/coala-bears/blob/master/bears/rest/RSTcheckBear.py)

Reply:
yes but  Ithink this is also fine as I have checked on google and found that it is fine

Reply:
But IMO to maintain consistency with coala-bears docs, RSTcheckBear will look better. Otherwise, rstcheckBear is looking fine